sdk/python/feast/infra/utils/postgres/postgres_config.py

```diff
@@ -0,0 +1,14 @@
+from typing import Optional
+
+from pydantic import StrictStr
+
+from feast.repo_config import FeastConfigBaseModel
+
+
+class PostgreSQLConfig(FeastConfigBaseModel):
+    host: StrictStr
+    port: int = 5432
+    database: StrictStr
+    db_schema: Optional[StrictStr] = None
```

Comment:
I think this should default to `"public"`
```suggestion
    db_schema: StrictStr = "public"
```

[As stated on Postgres's website](https://www.postgresql.org/docs/current/ddl-schemas.html#DDL-SCHEMAS-PUBLIC), the default schema name is "public".

> In the previous sections we created tables without specifying any schema names. By default such tables (and other objects) are automatically put into a schema named “public”.

Doing this also allows us to remove the `db_schema: StrictStr` declaration in `PostgreSQLOfflineStoreConfig`

Reply:
@achals, @felixwang9817 what are your thoughts on this one? I chatted with Gunnar about it (https://github.com/nossrannug/feast-postgres/pull/18#issuecomment-1068953104) and he said to refer to y'all

Reply:
Having it be public makes sense to me if that is the default!